emiliui.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="emiliui.css">
+    <title>Tado užduotis</title> 
+</head>
+<body>
+    <div class="container">
+        <fieldset>
+            <header class="header">
+                <div class="h1">
+                    <h1>Best <id class="blue">ToDo</id>  App</h1>
+                            <a href="https://www.facebook.com/"><i class="fa-brands fa-facebook"></i></a>
+                            <a href="https://www.twitter.com/"><i class="fa-brands fa-twitter"></i></a>
+                            <a href="https://www.linkedin.com/"><i class="fa-brands fa-linkedin"></i></a>
+                        </div>
+            </header>
+        </fieldset> <br>
+    </div>
+<main>
+    <div>
+        <form-id>
+        <section>
+            <div class="pirmas-tekstas">
+                <fieldset>
+                    <h2>Your Tasks</h2> 
+                        <label for="">
+                            <input type="text" value="e.g. wash dishes">
+                        </label>
+                        <button type="add">Add</button>
```

Comment:
Such button type does not exist. https://www.w3schools.com/tags/att_button_type.asp